syntax = "proto3";
package configuration;




import "collector_center_sdk/model/collector_center/configuration.proto";

/*
ListConfiguration请求
*/
message ListConfigurationRequest {
    /*
    页码
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 pageSize = 2 ;
    /*
    是否全部
    */
    bool isAll = 3 ;
}

/*
ListConfiguration返回
*/
message ListConfigurationResponse {
    /*
    页数
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 page_size = 2 ;
    /*
    总数
    */
    int32 total = 3 ;
    /*
    数据列表
    */
    repeated collector_center.Configuration list = 4 ;
}

/*
ListConfigurationApi返回
*/
message ListConfigurationResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    configuration.ListConfigurationResponse data = 4 ;
}

